src/na/na_ucx.c

```diff
@@ -924,6 +985,18 @@ na_ucx_initialize(na_class_t *nacl, const struct na_info NA_UNUSED *na_info,
         goto cleanup;
     }
 
+    if ((uctx_attrs.field_mask & UCP_ATTR_FIELD_THREAD_MODE) == 0) {
+        NA_LOG_ERROR("context attributes contain no thread mode");
+        ret = NA_PROTOCOL_ERROR;
+        goto cleanup;
+    }
+
+    /* Best effort to ensure thread safety */
+    if (context_thread_mode != UCS_THREAD_MODE_SINGLE
+        && uctx_attrs.thread_mode < context_thread_mode)
+        NA_LOG_WARNING("Context thread mode is: %s",
+            ucs_thread_mode_names[uctx_attrs.thread_mode]);
```

Comment:
Two-line statement -> open & close curly braces?

Reply:
it's technically still one line :) but anw this one should probably also fail in the same fashion so I'll add that too